cmake.sh

```diff
@@ -0,0 +1 @@
+cmake .. -DCMAKE_INSTALL_PREFIX=$PREFIX
```

Comment:
Was this script accidentally committed?  If you have a specific set of cmake options needed for a system, feel free to add a script to the `platforms` directory.  We could consider adding a `default.sh` script or similar.

Reply:
Oops! Removed the unnecessary file.